teddy.js

```diff
@@ -387,6 +370,39 @@
 
       renderedTemplate = teddy.templates[template] || renderedTemplate
 
+      // check if noparse or noteddy tag exist
+      noparse = renderedTemplate.match(/\snoparse/g)
+      noteddy = renderedTemplate.match(/\snoteddy/g)
+
+      // if 'noparse' or 'noteddy' attribute exists, set dontParse to true
+      if (noparse || noteddy) {
```

Comment:
Since noteddy/noparse could be matched in a single regex, that variable should be named `dontParse` which would obsolete this additional if statement.